client/src/pages/Landing.js

```diff
@@ -6,36 +6,36 @@ import { Route, Link } from "react-router-dom";
 
 import Ping from "./Ping";
 
-const landinPageStyle = theme => ({
+const landinPageStyle = (theme) => ({
   landingContainer: {
-    margin: theme.spacing.unit * 2
-  }
+    margin: theme.spacing.unit * 2,
+  },
```

Comment:
looks like your linter settings may be diff fro your teammates

Reply:
It is common to have a comma after the last property in react, but not in back end.